### ISSUE TITLE
Tensorrt 

### ISSUE THREAD

**Issue by toyal** [dbolya/yolact#116]
when i use the onnx model by tensorRT,get the error like follows,some suggestions。
----------------------------------------------------------------
While parsing node number 346 [Upsample]:
ERROR: builtin_op_importers.cpp:2026 In function importUpsample:
[8] Assertion failed: mode == "nearest"
[E] Failure while parsing ONNX file

段错误(吐核)

**Comment by dbolya** [comment]
This is not the place for issues with someone else's code. Message them directly, because I have no idea what's going on there (and they're likely not going to see this issue).

**Comment by rpaner** [comment]
Hello @toyal I had the same problem. It is caused because TensorRT does not support the mode 'bilinear' for the Upsample layer, only 'nearest' mode works.

To solve it just change the interpolation mode:
- In 'yolact.py' line 577 : `proto_downsampled = F.interpolate(proto_downsampled, size=outs[idx].size()[2:], mode='nearest')`
- In 'config.py' line 363: `'interpolation_mode': 'nearest',`

**Comment by toyal** [comment]
> This is not the place for issues with someone else's code. Message them directly, because I have no idea what's going on there (and they're likely not going to see this issu


> This is not the place for issues with someone else's code. Message them directly, because I have no idea what's going on there (and they're likely not going to see this issue).

sorry，i don‘t express the issues clearly，i transform the yolact model to onnx and then using  
 it by tensorrt. i solve the issues now.thx any more

**Comment by toyal** [comment]
> Hello @toyal I had the same problem. It is caused because TensorRT does not support the mode 'bilinear' for the Upsample layer, only 'nearest' mode works.
> 
> To solve it just change the interpolation mode:
> 
> * In 'yolact.py' line 577 : `proto_downsampled = F.interpolate(proto_downsampled, size=outs[idx].size()[2:], mode='nearest')`
> * In 'config.py' line 363: `'interpolation_mode': 'nearest',`

thanks,.but if i using "near" mode instead of "bilinear" then the accuracy  will be changed.what should i do for this.

**Comment by rpaner** [comment]
That's true, but I suggest you to try to change that configuration and see the results. It does not seem to degrade so much the outputs.
If you find another solution please write me.

**Comment by abhigoku10** [comment]
@toyal  can yo please let me know how did u convert yolact to onnx format ??
@rpaner  what is the fps you achieved when you use it using tensort ??

**Comment by abhigoku10** [comment]
@dbolya @toyal @rpaner  guys r u able to share the process of the conversion you have done , it would be helpful

**Comment by rpaner** [comment]
Sorry for not answering before @abhigoku10 I just converted the network to ONNX format thanks to https://github.com/dbolya/yolact/issues/74#issuecomment-513795638 repo changing the "bilinear" modes of upsample layer by "nearest" as I mentioned in a previous comment.

In terms of speed, I achieved 5 fps for resnet-50 architecture with an image input size of 550x550 in the Nvidia Jetson TX2.

**Comment by abhigoku10** [comment]
@rpaner  i am not able to generate the onnx model would you be able to share it for me for testing and as mentioned in #74 any other procedure you have followed

**Comment by poltavski** [comment]
@abhigoku10 @rpaner can you share conversion process to onnx, or at least, weights for pretrained model on coco/pascal sbd?

Any help will be much appreciated

**Comment by abhigoku10** [comment]
@poltavski can you share the link where i can upload the onnx model for you i shall share it sorry for the delayed response

**Comment by poltavski** [comment]
@abhigoku10 thank you for reply, could you use https://dropmefiles.com/ for this reason. I re-upload it on google drive, if somebody else needs it

Anything on conversion process, btw? I can convert model to onnx, but accuracy falls

**Comment by abhigoku10** [comment]
@poltavski  same observation the accuracy falls and there the confidence score of the predicted value is also low compared to without conversion

**Comment by simutisernestas** [comment]
@poltavski try this https://github.com/simutisernestas/yolact

**Comment by sdimantsd** [comment]
@simutisernestas what is the FPS on the jetson's (nano & xavier) ?

**Comment by simutisernestas** [comment]
@sdimantsd nano only ~3fps. Yesterday I've found this: https://github.com/haotian-liu/yolact_edge. It claims 30fps on Xavier :)

**Comment by sdimantsd** [comment]
@simutisernestas  Wow!!
I will check it, thanks!

**Comment by ynma-hanvo** [comment]
@toyal    hi, I could convert the yolact to onnx, but I am afraid of the post process in c++ code, 
can you share the code ,  really thankful for it.
